packages/plugins/sentry/test/sentry.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { defaultSkipError } from '../src/index.js';
+import { GraphQLError } from 'graphql';
+
+test('defaultSkipError should return true for GraphQLError', () => {
+  // Throwing a GraphQLError in GraphQL Yoga v3 results in error masking.
+  // The GraphQLError is treated as an expected error.
+  // The sentry plugin should follow this logic and do not capture GraphQLError.
+  expect(defaultSkipError(new GraphQLError('test', {}))).toBe(true);
```

Comment:
I don't think this behavior is correct as the graphql execution engine will wrap any error within `GraphQLError`.

Thus throwing a `GraphQLError` becomes a `GraphQLError` with an `originalError` property that is also a `GraphQLError`. In general, all errors whose `originalError` property is a `GraphQLError` is an expected errors that should be reported.

Throwing an non `GraphQLError` would result in a `GraphQLError` that has an `originalError` property of a non `GraphQLError` instance - which would not be reported.

Reply:
This should be used instead then;
https://github.com/n1ru4l/envelop/blob/main/packages/core/src/plugins/use-masked-errors.ts#L18

Reply:
it is more than this :D

Reply:
Why?

Reply:
Why?